great-idea/css/index.css

```diff
@@ -63,4 +63,196 @@ h1, h2, h3, h4, h5 {
     margin-bottom: 15px;
 }
 
-/* Copy and paste your work from yesterday here and start to refactor into flexbox */
\ No newline at end of file
+/* Copy and paste your work from yesterday here and start to refactor into flexbox */
+/* ----Main Layout Section---- */
+#main_container{
+	margin: 0 auto;
+	display: block;
+	width: 1200px;
+  }
+  #nav{
+	display: flex;
+	width: 1200px;
+	height: 80px;
+	align-items: center;
+  }
+  #heading{
+	display: flex;
+	width: 1200px;
+	height: 500px;
+	text-align: center;
+	vertical-align: middle;
+  }
+  #content_container{
+	display: flex;
+	width: 1200px;
+	height: 640px;
+	flex-wrap:wrap;
+  }
+  #contact_container{
+	display:  flex;
+	width: 1200px;
+	height: 300px;
+  }
+  #footer_container{
+	display:flex;
+	justify-content: center;
+	align-content: flex-end;
+	width: 1200px;
+	height: 200px;
+	text-align: center;
+  }
+  /* ----Content Control Section----- */
+  
+  /*Navigation Bar*/
+  nav{
+	display: flex;
+	align-items: center;
+	width: 70%;
+	justify-content: space-evenly;  
+  } 
+  #nav a{
+	padding:1%;
+	text-decoration: none;
+	color:rgb(66, 66, 66);
+	margin-left: 1.5%;
+	padding-bottom: 2%;
+  }
+  #nav img{
+	width: 15%;
+	margin-left: 13%;
+	padding-top: 3.5%;
+	align-self: flex-end;
+  }
+  
+  /*Heading Section*/
+  #heading_text{
+	display:flex;
+	width: 47%;
+	justify-content: center;
+	align-items: center;
+	flex-direction: column;
+  }
+  #heading_image{
+	width: 47%;
+	margin: 0 auto;
+	height: 500px;
+	padding: 70px 0;
+  }
+  #heading_text h1{
+	font-size: 50px;
+  }
+  #heading_text button{
+	background: white;
+	border: solid black;
+	padding: 15px;
+
+  } 
+  
+  
+  /*Content Section*/
+  .top_content{
+	display: inline-block;
+	width: 45%;
+	margin-left: 3%;
+	padding-top: 50px;
+	padding-bottom: 50px;
+  }
+  .content_image img{
+	display: inline-block;
+	width: 1200px;
+  }
+  .bottom_content{
+	display: inline-block;
+	width: 28%;
+	margin-left: 3%;
+	padding-top: 50px;
+	padding-bottom: 50px;
+  }
+
+
+   /*Services Section*/
+  .services_content{
+	  display: flex;
+	  flex-direction: column;
+	  justify-content: space-between;
+	  width: 47%;
+	  height: 200px;
+	  border: solid 1px black;
+	  background: lightgray;
+	  padding: 20px;
+  }
+
+  .services_content button{
```

Comment:
Good job using specificity to target elements rather than putting an ID or a class on everything - keep that up!